src/components/project-details/project-details.module.css

```diff
@@ -3,6 +3,45 @@
  * See the LICENSE file in the repository root folder for details.
  */
 
+#project_details {
```

Comment:
With the naming of the ID, does it impact the url you need to navigate to? 

I would prefer us to use kebab case for naming IDs:
This `<url>#project-details`
Instead of this `<url>#project_details`